git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH git-gud 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
```

Comment:
Can we capitalize "Get good..."

Reply:
No, as this is supposed to be all lowercase.

Reply:
No, as this is supposed to be all lowercase.